### DISCUSSION TITLE
An update of current macOS is required first

### DISCUSSION THREAD

**Discussion by Tanti0007** [grahampugh/erase-install#204] [Q&A] [answered]
Hi

Thanks very much for this project.

I have managed to successfully kick off the script, I have encountered a problem where the i get an error after Preboot files updates

"Unable to perform erase-install. An Update of the current MacOS is required first" 

Machine is an M1 2020, Big sur Macos Version 11.2.3, the version its trying to upgrade is MacOs 11.6.4

Is there anyway of ignoring this command and continue with the script ?

Thanks

**Comment by grahampugh** [comment, marked as answer]
No, if I remember correctly that was an error with some specific versions of macOS. You will indeed have to upgrade it before you can erase it, or fall back to Internet Recovery to wipe it.
